themes/digital.gov/layouts/partials/core/news.html

```diff
@@ -0,0 +1,24 @@
+<div class="dg-news__section">
```

Comment:
I am open to a different name for this class

Reply:
Could you also add some comments at the top of the file explaining what this does? Add an example link to a page that displays this component would be helpful.